Add-ons/UmbracoCourier/CommonIssues.md

```diff
@@ -6,21 +6,18 @@ _Document to outline commonly found issues with Courier and possible solutions_
 For all issues, always try these steps first:
 - Enable debugging in /config/courier.config and restart the app to see if courier has problems with loading dlls
 - Clear the cache folder in /app_data/courier/cache
-- Ensure you have the latest version, current 2.7.8.14
+- Ensure you have the latest version, current **3.1.7**
 
 ### Get the latest version here:
-Nightly.umbraco.org contains all builds, the current recommended version is **2.7.8.35** which contains all the latest 
-bug fixes for v4 and v6
+Nightly.umbraco.org contains all builds, the current recommended version is **3.1.7** which contains all the latest bug fixes.
 
-http://nightly.umbraco.org/UmbracoCourier/2.7.8/nightly%20builds/
+Access all the builds [here](http://nightly.umbraco.org/?container=umbraco-courier-release).
```

Comment:
Instead of using _here_ as a link, how about making the entire sentence the link? 🤔
```suggestion
[Access all the builds on Nightly Umbraco](http://nightly.umbraco.org/?container=umbraco-courier-release).
```